index.html

```diff
@@ -48,127 +47,92 @@ <h1>I'm Kenneth Glad to see you!</h1>
-              A daily selection of privately personalized: no accounts or
-              sign-ups required
-            </p>
-          </div>
-          <div>
-            <ul class="button">
-              <li><button type="button">html</button></li>
-              <li><button type="button">css</button></li>
-              <li><button type="button">javascript</button></li>
-            </ul>
-          </div>
-          <div class="hidden2"><button type="button">See Project</button></div>
+      <section id="works">
+        <div id="portfolio" class="works"></div>
+
+        <!-- desktop popup starts from Here -->
+
+        <div id="sectionWrap" class="modal">
+          <section class="desktop">
+            <div class="desktopIntro">
+              <h2 class="deskHeading"></h2>
+              <!-- <ul class="deskButtons">
+                <li class="deskList1"></li>
+                <li class="deskList2"></li>
+                <li class="deskList3"></li>
+              </ul> -->
+              
+              <div class="closeProj"><i class="fa fa-times" onclick="closeall()"></i></div>
+            </div>
+            <ul class="deskButtonlist"></ul>
+            <div class="imgContainer"><img src="images/desk1.svg" class="deskImag" alt="project-image"></div>
+            <div class="desktopFlex">
+              <p class="deskpara"></p>
+              <aside>
+                <div class="liner"></div>
+                <div class="separater">
+                  <div class="seeLive">
+                    <a href="" id="liveLink"> see Live <img src="images/live.svg" alt="github pages"> </a>
+                  </div>
+                  <div class="seeSource">
+                    <a href="" id="sourceLink"> See Source <img src="images/Github.svg" alt="github"> </a>
+                  </div>
+                </div>
+              </aside>
+            </div>
+          </section>
         </div>
-        <div class="tonic">
-          <img src="images/Snapshoot-Portfolio-4.svg" alt="snap4" />
-          <h2 id="stories-header2">Multi-post stories</h2>
-          <ul id="profs3">
-            <li id="cannopy3">CANOPY</li>
-            <li>Back End Dev</li>
-            <li>2015</li>
-          </ul>
-          <p>
-            A daily selection of privately personalized: no accounts or sign-ups
-            required
-          </p>
-          <ul class="button">
-            <li><button type="button">html</button></li>
-            <li><button type="button">css</button></li>
-            <li><button type="button">javascript</button></li>
-          </ul>
-          <div class="hidden3">
-            <button type="button">See Project</button>
-          </div>
+        <!-----desktop popup ends here---------->
+
+        <!-- mobile popup starts from here -->
+
+        <div id="mobileWrap" class="modal">
+          <section class="mobileDetail">
+            <div class="mobileIntro">
+              <div class="top-Modal">
+                <div>
```

Comment:
- [ ]  Kindly try to make the mobile design look as much as possible like it is in Figma. the placement of the modal elements is different here too and the technologies badges look very different from the ones in Figma ✍️ 

## Your design:
![image](https://user-images.githubusercontent.com/50721479/190478526-0cf4b29f-d0a1-46cb-b603-e7995b5d0efc.png)

## Figma design:
![image](https://user-images.githubusercontent.com/50721479/190478598-9407994f-08ac-4c4c-860f-2c03e408149e.png)